### ISSUE TITLE
Assign assets from a collection to tags not having any assets assigned yet

### ISSUE THREAD

**Issue by pduersteler** [neos/neos-development-collection#2150]
### Description

As a content editor, I want to assign assets filtered by a collection via drag and drop to multiple tags.

### Steps to Reproduce

1. Navigate to the media library
2. Ensure to have a tag that has no assets assigned to it
3. Select an asset collection that has assets in it
4. The tag is not visible to drop an asset on it

#### Expected behavior

All tags are visible when a collection is selected, or should appear when an asset is being dragged an dropped to be able to assign an asset to a unused tag.

Side note: When searching for an asset, all tags and collections show up, as expected.

#### Actual behavior

Tags without any assets associated won't show up when a collection is selected. This makes it quite tedious to assign tags to assets.

### Affected Versions

Neos: 3.3.x and 4.0.x

Flow: 4.3.x and 5.0.x

**Comment by Nikdro** [comment]
You have to edit the Collection and enable Tags for that collection. Afterwards, you will get the Tags (also the unused) in the overview.

It's a feature, not a bug. Right?

**Comment by kdambekalns** [comment]
Indeed, tags are "per collection". Even though it's a bit inconsistent to be able to have an asset with a tag in a collection that does not "support" that tag. 🤷‍♂️
